CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+## Setup for Code Contributions
+
+### Prerequisites
+
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`
+- Install Node.js >= 16
+
+##### Windows
+- Install current version of Python3
+- Install Visual Studio Code and add the Microsoft C/C++ extension
+- Install Node.js >= 16
+
+##### Linux
+
+- Install Python > 3.7
+- Install `make`, `gcc`, and `g++`
+- Install Node.js >= 16
+
+#### Repository Setup
+- Fork the project.
+- Open a Terminal and navigate to where you want the project repository to exist
+- Clone your forked repository
+    ```sh
+    git clone https://github.com/<your-github-user>/club_website.git
+    ```
+- Set the upstream remote
+    ```sh
+    git remote add upstream https://github.com/NSC-Computer-Science-Club/club_website.git
+    ```
+- Verify
+    ```sh
+    git remote -v
+    ```
+- Navigate into the root directory
+    ```sh
+    cd club_website
```

Comment:
Ah, there's the cd. This should probably be moved up to right below the clone. Maybe even in the same code block.

Reply:
oops yea that was part of the reorder